packages/react-theming/README.md

```diff
@@ -365,11 +376,33 @@ downlevel slots.
 
 ### Slots
 
-TODO: Describe how to interact with slots
+`props.slots` will be a dictionary mapping individual slots to
+`ReactType` values which can then be used in JSX.
+
+`slots` are defined in createComponent by specifying a slots argument:
+
+```jsx
+const CreatedComponent = createComponent(BaseComponent, {
```

Comment:
Is it going to be `createComponent` or `compose`? I see both terms being used at different times in this PR.

Reply:
Ahh, good call. I meant to keep this consistent. It's `compose` now, but we are going to change it as several FP libraries already have a `compose` impl that differs.